### DISCUSSION TITLE
Will this screen work?

### DISCUSSION THREAD

**Discussion by rradomir** [Bodmer/TFT_eSPI#3103] [Q&A - General]
Question is simple as in title.
[Link for screen](https://aliexpress.com/item/1005005958054196.html?spm=a2g0o.store_pc_allItems_or_groupList.0.0.671822caktsGeo&pdp_npi=4%40dis%21PLN%2140%2C47%20z%C5%82%2140%2C51%20z%C5%82%21%21%21%21%21%40211b600917052652899998242ef63a%2112000035042805010%21sh%21PL%212038545549%21&gatewayAdapt=glo2pol)

> Driver ic:ILI9488
> 
> Interface: mcu i8080 8bit/16bit,spi-3wire/4-wire

I think this should work, i have other screen with spi 3-4wire and ili9488 driver (on rpi pico)
Im interested in module with touch input. I asked seller, it is XPT2046

I want to use it with rpi pico or esp32 :) preffered esp32.

**Comment by Bodmer** [comment]
The ILI9488 is supported with 8bit parallel and ESP32. You will need an independant touch library as tough is not supported in parallel mode.

**Comment by rradomir** [comment]
Sorry for very late answer, but I like to close topics, when I got solution. Maybe someone also will be wondering.
So - you are partially wrong. Your library works with touch and display on this screen, but MISO (SDO) from screen must be disconnected.
Tested successfully on ESP32. Didn't tried on pico.

**Reply by solidstate90** [reply]
How did you set up your User_Setup.h file to get touch working with the ili9488 + xpt2046??

**Reply by rradomir** [reply]
> How did you set up your User_Setup.h file to get touch working with the ili9488 + xpt2046??

Well, link for screen is broken now, so i need to guess, which screen it was. But looking on sketch I don't see anything unusual. I use UserSetup21 and added:
`#define TOUCH_CS 12`
also `TFT_MISO `is disconnected and not defined (but maybe it's still defined somewhere else in library to some defaults).
